### PR TITLE
add a configuration feature which configures the JSON object mapper

### PR DESCRIPTION
**Pull request by warrenc5**
for example

<features>
    <feature>com.fasterxml.jackson.databind.SerializationFeature.WRITE_ENUMS_USING_TO_STRING</feature>
</features>

### CODE REVIEW

**Comment by Postremus** [comment]
I would personally prefer if we could just define some sort of ```<objectMapper>some.package.ObjectMapperFactory<objectMapper>``` element in the configuration.
The ObjectMapperFactory could in this way return an ObjectMapper, which is setup as needed by the developer.

@who What is your opinion on this topic?

**Comment by warrenc5** [comment]
we share this plugin configuration across multiple modules, and I was looking for a way without requiring any *Java* coding from the user.

**Comment by warrenc5** [comment]
Ok, it's tested with one success and one fail case.

File: README.md [hunk 1]
```diff
@@ -75,7 +75,8 @@ Import the plugin in your project by adding following configuration in your `plu
 | `jsonExampleValues` | If `true`, all example values in `@ApiModelProperty` will be handled as json raw values. This is useful for creating valid examples in the generated json for all property types, including non-string ones. |
 | `modelConverters` | List of custom implementations of `io.swagger.converter.ModelConverter` that should be used when generating the swagger files. | 
 | `swaggerExtensions` | List of custom implementations of `io.swagger.jaxrs.ext.SwaggerExtension` that should be used when generating the swagger files. | 
-
+| `enabledFeatures`    | List of ConfigFeature enums that are supported by ObjectMapper.configure - the feature is set to true. https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#features) here, see more details [below](#features)|
```
Comment (who) [inline README.md]: This should be `enabledObjectMapperFeatures`.

File: README.md [hunk 2]
```diff
@@ -75,7 +75,8 @@ Import the plugin in your project by adding following configuration in your `plu
...
+| `disabledFeatures`    | List of ConfigFeature enums that are supported by ObjectMapper.configure - the feature is set to false. https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#features) here, see more details [below](#features)|
```
Comment (who) [inline README.md]: This should be `disabledObjectMapperFeatures`.

**Comment by who** [comment]
@warrenc5  Thanks for making the changes. It appears that there is a merge conflict with the PR. Could you please resolve this?

**Comment by who** [comment]
@warrenc5 Thanks for resolving the conflicts. I'm going to merge. 

Your PR is appreciated!

**Comment by scara** [comment]
TNX @warrenc5 for your hard working! 😄 
TNX @who for your merge! 👍 

This is a nice and important feature for those using enums: when will 3.1.8 be officially released?

TIA,
Matteo

**Pull request merged**